Docs/Tutorial.md

```diff
@@ -0,0 +1,146 @@
+# Generating an API with CreateAPI
+
+In this tutorial, you are going to create a new library using CreateAPI that talks to the [Swagger Petstore](https://petstore3.swagger.io/) API.
+
+- [Getting Started](#getting-started)
+- [Generating a Package](#generating-a-package)
+- [Using the Code](#using-the-code)
+- [Adding a Configuration File](#adding-a-configuration-file)
+- [Updating the Code](#updating-the-code)
+- [Next Steps](#next-steps)
+
+## Getting Started
+
+If you haven't already heard about CreateAPI, it's a tool that allows you to generate Swift code from an OpenAPI specification.
+
+For a new project, the easiest way to get started with CreateAPI is to generate the entire package using the `--package` option. Once you have your package in place, you can build upon it yourself and if you need to update the generated entities or paths later on, you can do so using the `--module` option to avoid overwriting the rest of your changes.
+
+If you haven't already, be sure to [install](../README.md#installation) the `create-api` CLI.
+
+## Generating a Package
+
+First things first, you need to generate the package. To do so, there is one key ingredient: The schema. Download the Petstore schema from [https://petstore3.swagger.io/api/v3/openapi.json](https://petstore3.swagger.io/api/v3/openapi.json) and save it somewhere accessible on your computer.
+
+Using terminal, run the `create-api generate` command and specify the following:
+
+1. The `--package` option, followed by the name of your package
+2. The `--output` option, followed by the directory that you want to write the package to
+3. The path to the schema that you just downloaded
+
+For example:
+
+```bash
+$ create-api generate --package "PetstoreKit" --output "./" "path/to/the/downloaded/schema.json"
+```
+
+You will see something like the following printed to the console:
+
+> Generating code for schema.json...
+
+And you will find a new folder called **PetstoreKit** located within the output directory. Double click on the **Package.swift** file within the package to open it in Xcode and you will see something like the following:
+
+<img
+  src="./Images/Tutorial_01_NewPackage.png#gh-light-mode-only"
+  alt="A screenshot of Xcode focusing on the Paths.swift source file within the newly generated PetstoreKit package. To the left, you can see the project navigator which displays the source files (Entites.swift, Paths.swift) and the resolved dependencies (Get, HTTPHeaders, URLQueryEncoder)."
+/><img
+  src="./Images/Tutorial_01_NewPackage_Dark.png#gh-dark-mode-only"
+  alt="A screenshot of Xcode focusing on the Paths.swift source file within the newly generated PetstoreKit package. To the left, you can see the project navigator which displays the source files (Entites.swift, Paths.swift) and the resolved dependencies (Get, HTTPHeaders, URLQueryEncoder)."
+/>
```

Comment:
I went all out on this 😅 

https://user-images.githubusercontent.com/482871/182634333-0d1d78f2-5b50-40a2-b197-b1c4570fc55c.mp4

https://github.blog/changelog/2021-11-24-specify-theme-context-for-images-in-markdown/

Reply:
Love this.